src/main/kotlin/io/bitvavo/simulation/formatters/OrderBookFormatter.kt

```diff
@@ -0,0 +1,38 @@
+package io.bitvavo.simulation.formatters
+
+import io.bitvavo.simulation.models.OrderBookSnapshot
+
+object OrderBookFormatter {
+    fun formatOrderBook(orderBook: OrderBookSnapshot): String {
```

Comment:
You can also use https://kotlinlang.org/docs/extensions.html for this or make it part of the class

Reply:
I know, but I wanted an explicit entity instead of extension here.